package.json

```diff
@@ -3999,6 +3999,446 @@
+														"command": "language-julia.chooseModule"
+													},
+													{
+														"key": "p",
+														"name": "Open package directory",
+														"icon": "new-folder",
+														"type": "command",
+														"command": "language-julia.openPackageDirectory"
+													},
+													{
+														"key": "t",
+														"name": "Tag new package version (experimental)",
+														"icon": "tag",
+														"type": "command",
+														"command": "language-julia.tagNewPackageVersion"
+													},
+												]
+											},
+											{
+												"key": "=",
+												"name": "+Format",
+												"icon": "list-flat",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "=",
+														"name": "Format region or buffer",
+														"icon": "list-flat",
+														"type": "command",
+														"command": "editor.action.format"
+													},
+													{
+														"key": "b",
+														"name": "Format buffer",
+														"icon": "file",
+														"type": "command",
+														"command": "editor.action.formatDocument"
+													},
+													{
+														"key": "c",
+														"name": "Format changes",
+														"icon": "diff",
+														"type": "command",
+														"command": "editor.action.formatChanges"
+													},
+													{
+														"key": "s",
+														"name": "Format selection",
+														"icon": "selection",
+														"type": "command",
+														"command": "editor.action.formatSelection"
+													},
+													{
+														"key": "B",
+														"name": "+Format buffer with formatter",
+														"icon": "file",
+														"type": "command",
+														"command": "editor.action.formatDocument.multiple"
+													},
+													{
+														"key": "S",
+														"name": "+Format selection with formatter",
+														"icon": "selection",
+														"type": "command",
+														"command": "editor.action.formatSelection.multiple"
+													}
+												]
+											},
+											{
+												"key": "b",
+												"name": "+Backend",
+												"icon": "circuit-board",
+												"type": "bindings",
+												"bindings": [
+													{
+														"key": "i",
+														"icon": "output",
+														"type": "command",
+														"command": "language-julia.refreshLanguageServer",
+														"name": "Julia: Re-Index Language Server Cache",
+													},
+													{
+														"key": "r",
+														"icon": "server-process",
+														"type": "issue-reopened",
+														"command": "language-julia.restartLanguageServer",
+														"name": "Julia: Restart Language Server",
+													},
+													{
+														"key": "R",
+														"name": "Re-index language server cache",
```

Comment:
Seems like this is the same as "i" binding above?

Reply:
I ran all of the `=` and `g` actions. I removed two related to formatting because there is only one formatter currently.

Reply:
Thanks for double checking. I would suggest to leave those two formatting bindings in because a user might use other formatter that's not from Julia extension (although that's probably rare)